pkgs/top-level/all-packages.nix

```diff
@@ -9312,6 +9312,20 @@ in
     ruby_2_5
     ruby_2_6;
 
+  rubyMinimal = ruby.override {
+    useRailsExpress = false;
+    rubygemsSupport = false;
+    zlibSupport = false;
+    opensslSupport = false;
```

Comment:
What are you using ruby for that does not support. HTTPS/HTTP with deflate support?

Reply:
It’s about the minimal runtime closure, not so much about supporting openssl.

Reply:
But if http clients are a thing most small ruby scripts would like to use, we should enable it.

For example the [home-manager service activation script](https://github.com/rycee/home-manager/blob/master/modules/systemd-activate.rb) doesn’t need SSL.